src/ir/field_selector_test.cc

```diff
@@ -0,0 +1,62 @@
+#include "src/ir/field_selector.h"
+
+#include <string>
+
+#include "absl/hash/hash_testing.h"
+#include "src/common/testing/gtest.h"
+
+namespace raksha::ir {
+
+static std::string sample_field_names[] = {
```

Comment:
Is this the idiomatic way to define parameters for `TestWithParam`?

Reply:
From what I can see, it doesn't look that there is one prescribed, most-correct way to use `TestWtihParam`. This looks like a pattern that is used at least some of the time.